app/components/joyride-foundation.js

```diff
@@ -43,6 +54,10 @@ export default Ember.Component.extend({
         function showJoyrideOverlay(){
           Ember.$(".joyride-view-background").show();
         }
+
+        function setSeenTour(){
+          _this.set("session.seenTour", _this.get("session.seenTour") + "," + _this.get("currentUserId"));
```

Comment:
@steveyken yes, previously I did try it with only `currentUser`.
But there comes a case, 
- Donor logged in and created offer, seen joyride-tour, now deleted offer. And logs out.
- Again Donor logged in with another user and followed same steps as above
- Again Donor logged in with his first login(mobile) and creates new offer, here he should not see the joyride
(Not sure, if this is a obvious case or if Donor uses different browser, he would see it again)
Let me know if you have any thoughts here.